### ISSUE TITLE
FEAT: Make macOS app icon a rounded-rectangle to match Big Sur style guide

### ISSUE THREAD

**Issue by rakleed** [beekeeper-studio/beekeeper-studio#1741]
**Is your feature request related to a problem? Please describe.**
The macOS app icon stands out a lot compared to other app icons. I have to manually change the icon each time to match the HIG.

**Describe the solution you'd like**
Could you please update the app icon for macOS according to Big Sur specifications?

**Describe alternatives you've considered**

**Additional context**

![CleanShot 2023-10-15 at 17 55 02](https://github.com/beekeeper-studio/beekeeper-studio/assets/19418601/2bfd52c4-57e6-4e89-ac53-7fdf9069a50e)

More: https://developer.apple.com/design/human-interface-guidelines/app-icons#Platform-considerations

Production Templates: https://developer.apple.com/design/resources/#macos-apps

**Database type, if appropriate**

**Comment by DAcodedBEAT** [comment]
@beekeeper-studio I think this should be escalated now that macOS Tahoe 26 is generally available. The Beekeeper Studio icon sticks out like a sore thumb.

<img width="360" height="270" alt="Image" src="https://github.com/user-attachments/assets/785d8816-7344-4b44-a955-987e6646710e" />

**Comment by gabriel-gof** [comment]
Yes, this should receive attention. It's very ugly the on Tahoe.

**Comment by gabriel-gof** [comment]
no one really cares about that?

**Comment by rathboma** [comment]
You are right, we should do this. I just bought @azmy60 a Mac to help us catch things like this.

**Comment by DAcodedBEAT** [comment]
@rathboma any progress? I realized this issue is still present
